dev/tests/api-functional/testsuite/Magento/GraphQl/DownloadableProduct/AddDownloadableProductToCartTest.php

```diff
@@ -0,0 +1,337 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\DownloadableProduct;
+
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface;
+use Magento\Quote\Model\ResourceModel\Quote as QuoteResource;
+
+/**
+ * Class AddDownloadableProductToCartTest
+ */
+class AddDownloadableProductToCartTest extends GraphQlAbstract
+{
+    /**
+     * @var QuoteResource
+     */
+    private $quoteResource;
```

Comment:
Need to remove redundant dependencies and make refactoring of tests

Looks at the set of fixtures:
https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files

Examples of using:
https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer

Thanks

Reply:
**done**